domains/Contact/ManageContactFeed/Web/ViewHelpers/ModuleFeedViewHelper.php

```diff
@@ -17,16 +17,14 @@ class ModuleFeedViewHelper
 {
     public static function data($items, User $user, Vault $vault): array
     {
-        $itemsCollection = $items->map(function ($item) use ($user) {
-            return [
-                'id' => $item->id,
-                'action' => $item->action,
-                'author' => self::getAuthor($item),
-                'sentence' => self::getSentence($item),
-                'data' => self::getData($item, $user),
-                'created_at' => DateHelper::format($item->created_at, $user),
-            ];
-        });
+        $itemsCollection = $items->map(fn ($item) => [
```

Comment:
We don't need a `use ($user)` with this?

Reply:
That's the power of the `fn` https://www.php.net/manual/fr/functions.arrow.php